doc/classes/HScrollBar.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="HScrollBar" inherits="ScrollBar" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Horizontal scroll bar.
+		A horizontal scrollbar that goes from left (min) to right (max).
 	</brief_description>
 	<description>
-		Horizontal version of [ScrollBar], which goes from left (min) to right (max).
+		A horizontal scrollbar, used to navigate through content that extends beyond the visible width of a control. It is a [Range]-based control and goes from left (min) to right (max).
```

Comment:
```suggestion
		A horizontal scrollbar, typically used to navigate through content that extends beyond the visible width of a control. It is a [Range]-based control and goes from left (min) to right (max).
```
what I suggested in my other comment
(same in VScrollBar I assume)

Reply:
OK, I addressed both.